src/scanner/mod.rs

```diff
@@ -11,9 +11,12 @@ use futures::stream::FuturesUnordered;
 use std::{
     io::ErrorKind,
     net::{IpAddr, Shutdown, SocketAddr},
+    num::NonZeroU8,
     time::Duration,
 };
 
+use super::warning;
```

Comment:
Please merge this with `use super::PortStrategy;`

Reply:
I added it locally.